src/main/bash/commands/kobman-install.sh

```diff
@@ -50,31 +50,29 @@ function __kobman_create_environment_directory
 
 		local environment_name=$1
 		local version_id=$2
+		mkdir -p ${KOBMAN_DIR}/envs/kob_env_"${environment_name}"
+		touch ${KOBMAN_DIR}/envs/kob_env_${environment_name}/current
+		current="${KOBMAN_DIR}/envs/kob_env_${environment_name}/current"
 
-		cd "${KOBMAN_DIR}/envs"
-		mkdir -p kob_env_"${environment_name}"
-		cd kob_env_"${environment_name}"
-		touch current
-		destdir="${KOBMAN_DIR}/envs/kob_env_${environment_name}/current"
-
-		if [ ! -d "$version_id" ]
+		if [[ ! -f ${KOBMAN_DIR}/envs/kobman-${environment_name}.sh ]]; then
+			__kobman_echo_debug "Could not find file kobman-$environment_name.sh"
+			return 1
+		fi
+		if [[ ! -d ${KOBMAN_DIR}/envs/kob_env_${environment_name}/$version_id ]];
 		then
-			mkdir -p $version_id
-			cd $version_id                                          # Needs to be refactored identify the latest version
-			__kobman_echo_no_colour "$version_id" > "$destdir"
-			cp "${KOBMAN_DIR}/envs/kobman-${environment_name}.sh" .
+			mkdir -p ${KOBMAN_DIR}/envs/kob_env_${environment_name}/$version_id
```

Comment:
the format of kob_env_{environment_name} is used here , whereas you are using kobman-{environmen_name} is this intentionally maintained as separate. can the prefix be same. kobman- and kob_env

Reply:
It was intentionally kept. kob_env_ is the folder which gets created during installaton under .kobman. Can be made same.
Then we will have to change the references of that name in other codes as well